mods/cnc/weapons/missiles.yaml

```diff
@@ -30,136 +30,78 @@ Rockets:
 	Warhead@3Eff: CreateEffect
 		Explosions: small_frag
 		ImpactSounds: xplos.aud
+		TargetSearchRadius: 0
+
+Rockets:
+	Inherits: ^MissileWeapon
 
 BikeRockets:
-	ReloadDelay: 50
-	Range: 6c0
-	Report: bazook1.aud
-	ValidTargets: Ground, Air
+	Inherits: ^MissileWeapon
 	Burst: 2
 	BurstDelay: 10
 	Projectile: Missile
-		Arm: 0
-		Blockable: false
-		Inaccuracy: 128
-		Image: DRAGON
 		HorizontalRateOfTurn: 10
-		TrailImage: smokey
-		ContrailLength: 8
 		Speed: 213
-		RangeLimit: 7c204
 	Warhead@1Dam: SpreadDamage
 		Spread: 128
 		Damage: 30
-		ValidTargets: Ground, Air
 		Versus:
 			None: 25
 			Wood: 75
-			Light: 100
-			Heavy: 100
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
-	Warhead@3Eff: CreateEffect
-		Explosions: small_frag
-		ImpactSounds: xplos.aud
 
 OrcaAGMissiles:
+	Inherits: ^MissileWeapon
 	ReloadDelay: 12
 	Burst: 2
 	BurstDelay: 12
 	Range: 5c0
 	MinRange: 1c256
-	Report: bazook1.aud
 	ValidTargets: Ground
 	Projectile: Missile
 		Arm: 1
-		Blockable: false
-		Inaccuracy: 128
-		Image: DRAGON
 		HorizontalRateOfTurn: 20
-		TrailImage: smokey
-		ContrailLength: 8
 		Speed: 256
 		RangeLimit: 6c0
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 25
 		ValidTargets: Ground
 		Versus:
 			None: 50
 			Wood: 100
 			Light: 100
 			Heavy: 75
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
-	Warhead@3Eff: CreateEffect
-		Explosions: small_frag
-		ImpactSounds: xplos.aud
 
 OrcaAAMissiles:
-	ReloadDelay: 12
-	Burst: 2
-	BurstDelay: 12
-	Range: 5c0
-	MinRange: 1c256
-	Report: bazook1.aud
+	Inherits: OrcaAGMissiles
```

Comment:
> This changes the None versus from 100% to 50%, but i'm pretty sure that doesn't matter in practice (we don't have any unarmored flying actors).

Yeah, I intentionally didn't bother 'fixing' it for that reason.

> Smudges are spawned on ground even when the weapon explodes on an aircraft

Ok, I didn't expect/know that, I'll fix it.